src/components/shell-panel/shell-panel.tsx

```diff
@@ -47,9 +49,31 @@ export class ShellPanel implements ConditionalSlotComponent, LocalizedComponent,
 
   /**
    * When `true`, the content area displays like a floating panel.
+   *
+   * @deprecated use `displayMode` instead.
    */
   @Prop({ reflect: true }) detached = false;
 
+  @Watch("detached")
+  handleDetached(value: boolean): void {
+    if (value) {
+      this.displayMode = "detached";
+    } else if (this.displayMode === "detached") {
+      this.displayMode = "docked";
+    }
+  }
+
+  /**
+   * Specifies the display mode - `"docked"` (full height, displays adjacent to center content), `"detached"` (not full height, content separated detached from action bar, displays on top of center content),
```

Comment:
+@geospatialem for editorial consideration

Reply:
nitpick:

> .....content is separated from `calcite-action-bar`, displays....